File: app/src/main/java/com/pgshare/studentroomsharingapp/Adapter/Owner.java
package com.pgshare.studentroomsharingapp.Adapter;

public class Owner {

    private String ownerId, ownerName, ownerEmail, ownerPhone, userType,imageUrl, gender;

    public Owner(String ownerId, String ownerName, String ownerPhone, String gender, String imageUrl, String userType) {
        this.ownerId = ownerId;
        this.ownerName = ownerName;
        this.ownerPhone = ownerPhone;
        this.userType = userType;
        this.gender = gender;
        this.imageUrl = imageUrl;
    }

    public Owner(String ownerId, String ownerEmail) {
        this.ownerId = ownerId;
        this.ownerEmail = ownerEmail;
    }


    public String getUserType() {
        return userType;
    }

    public void setUserType(String userType) {
        this.userType = userType;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    public String getOwnerId() {
        return ownerId;
    }

    public void setOwnerId(String ownerId) {
        this.ownerId = ownerId;
    }

    public String getOwnerName() {
        return ownerName;
    }

    public void setOwnerName(String ownerName) {
        this.ownerName = ownerName;
    }

    public String getOwnerEmail() {
        return ownerEmail;
    }

    public void setOwnerEmail(String ownerEmail) {
        this.ownerEmail = ownerEmail;
    }

    public String getOwnerPhone() {
        return ownerPhone;
    }

    public void setOwnerPhone(String ownerPhone) {
        this.ownerPhone = ownerPhone;
    }
}

File: app/src/main/java/com/pgshare/studentroomsharingapp/Adapter/Owner.kt
package com.pgshare.studentroomsharingapp.Adapter

class Owner {
    var ownerId: String? = null
    var ownerName: String? = null
    var ownerEmail: String? = null
    var ownerPhone: String? = null
    var userType: String? = null
    var imageUrl: String? = null
    var gender: String? = null

    constructor()

    constructor(
        ownerId: String?,
        ownerName: String?,
        ownerPhone: String?,
        gender: String?,
        imageUrl: String?,
        userType: String?
    ) {
        this.ownerId = ownerId
        this.ownerName = ownerName
        this.ownerPhone = ownerPhone
        this.userType = userType
        this.gender = gender
        this.imageUrl = imageUrl
    }

    constructor(ownerId: String?, ownerEmail: String?) {
        this.ownerId = ownerId
        this.ownerEmail = ownerEmail
    }

    constructor(ownerId: String?, ownerName: String?, ownerEmail: String?, ownerPhone: String?) {
        this.ownerId = ownerId
        this.ownerName = ownerName
        this.ownerEmail = ownerEmail
        this.ownerPhone = ownerPhone
    }
}
